config.txt

```diff
@@ -93,8 +93,15 @@ null,
  * pool_address	  - Pool address should be in the form "pool.supportxmr.com:3333". Only stratum pools are supported.
  * wallet_address - Your wallet, or pool login.
  * pool_password  - Can be empty in most cases or "x".
+ *
+ * Featured small pools       fee    
+ * pool.minemonero.pro:3333 - 0.9%
+ * pool.usxmrpool.com:3333  - 0.5%
+ * pool.xmr.pt:3333         - 0.4%
+ *
+ * We feature pools up to 1MH/s here. For a more complete list see M5M400's pool list at www.moneropools.com
  */
-"pool_address" : "pool.supportxmr.com:3333",
+"pool_address" : "pool.usxmrpool.com:3333",

```

Comment:
Pool hopping is a real problem - you can throw 1MH/s from nicehash for an hour and collect most of the reward from the pool when they eventually find a block. Of course the proper fix is technical (PPLNS), do you think this should disqualify them?